src/scenes/UserGalleryPage/UserGalleryCollections.tsx

```diff
@@ -115,19 +98,69 @@ function UserGalleryCollections({ galleryRef, queryRef, mobileLayout }: Props) {
     return <EmptyGallery message={emptyGalleryMessage} />;
   }
 
+  const rowRenderer = ({ index, key, parent, style }: ListRowProps) => {
```

Comment:
nit: wrap this in `useCallback` just to be safe. i can make the commit since i have the file open!

Reply:
Thanks @Robinnnnn  for the update 🙏🏻